.github/workflows/deploy.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches:
       - main
+
+concurrency:
+  group: "deploy"
+  cancel-in-progress: true
```

Comment:
Do we want to cancel the in progress? Or is it better to let it complete and the next to queue up?

Reply:
You know, you're probably right, it's probably better to leave the default pending behavior! I'll swap to that, thanks!